test/__init__.py

```diff
@@ -852,6 +852,10 @@ def max_bson_size(self):
     def max_write_batch_size(self):
         return (self.hello)["maxWriteBatchSize"]
 
+    @property
+    def max_message_size_bytes(self):
```

Comment:
What's the motivation for this, reducing code duplication? I like it a lot.

Reply:
Yep! I wanted to avoid the call to `client_context.hello` everytime one of the tests depended on the `max_message_size_bytes` value

Reply:
Do any tests change this value?

Reply:
None of the tests change the `client_context.max_message_size_bytes` value itself, they just use the value for other calculations